pandas/tests/test_col.py

```diff
@@ -31,6 +32,8 @@
         (pd.col("a") < 1, [False, False], "(col('a') < 1)"),
         (pd.col("a") <= 1, [True, False], "(col('a') <= 1)"),
         (pd.col("a") == 1, [True, False], "(col('a') == 1)"),
+        (np.log(pd.col("a")), [0.0, 0.6931471805599453], "log(col('a'))"),
```

Comment:
`log` is dangerous to use in a test because the floating point value could be different on different platforms.  

Maybe use `np.min()` instead.

Reply:
sure, thanks! have replaced